.github/workflows/ubuntu-ci-c6a-x86_64.yaml

```diff
@@ -0,0 +1,210 @@
+name: ubuntu-ci-c6a-x86_64-build
```

Comment:
let's rename this to `ubuntu-ci-c6a-x86_64-gcc-build` (same in line 20)

I will add a separate version for Intel, and later we can add one for oneAPI